### PR TITLE
Lava map slights

### LINKED ISSUE #35092: The three numbers in this atmos warning dialogue are not zero, and have not been zero for a long time.
Round ID: 82727 and many others

This message starts with a allcaps "HEY! LISTEN!" for a reason. This needs to be fixed.

### LINKED ISSUE #35190: Lavaland Syndicate Base doesn't have FLOUR
How to reproduce: Try to cook anything other than monkey steak in the lavabase kitchen. Note that it has milk and eggs. I was very upset when I couldn't make braincake from other ops committing suicide.

### PR DESCRIPTION
**Pull request by Jalleo**
fixes #35092 (At least with the tiles to the north not sure if the walls needed this or not
fixes #35190 
edit. I did add a few no lava effects on some ruins and a TGM pass again removed some keys so I added them in.

### CODE REVIEW

File: _maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm [hunk 1]
```diff
@@ -96,9 +96,6 @@
 /obj/effect/decal/cleanable/dirt,
 /turf/open/floor/plasteel/floorgrime,
 /area/ruin/unpowered/syndicate_lava_base/chemistry)
-"cO" = (
-/turf/closed/wall/mineral/plastitanium/explosive,
```
Comment (Jalleo) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm]: uuh are these meant to stay around?

Comment (DaedalusGame) [reply _maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm]: yes. these are what makes the self destruct work.

Comment (ShizCalev) [reply _maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm]: assuming you used mapmerge2, it's fine. they're just unused keys being removed from the dictionary.

Comment (Jalleo) [reply _maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm]: I did use mapmerge2

Comment (praisenarsie) [reply _maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm]: tis fine then

**Pull request merged**